lib/oneshot_coverage.rb

```diff
@@ -22,17 +22,19 @@ def call(env)
   OneshotLog = Struct.new(:path, :md5_hash, :lines)
 
   class Reporter
-    def initialize(target_path:, logger:, emit_term: nil)
+    def initialize(target_path:, logger:, emit_term: nil, check_bundle_path:)
```

Comment:
Don't add keyword argument not having default value after keyword argument with default value.
I want the `initialize` method interface to be backward compatible, so in this case, `false` may be good default value.

Reply:
Ok. I'll make sure to include the default value.